refarch-frontend/src/views/HomeView.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-container>
+    <v-row class="text-center">
+      <v-col cols="12">
+        <v-img
+          src="@/assets/logo.png"
+          class="my-3"
+          height="200"
+        />
+      </v-col>
+
+      <v-col class="mb-4">
+        <h1 class="text-h3 font-weight-bold mb-3">
+          Willkommen beim RefArch-Kickstarter
+        </h1>
+        <p>
+          Das API-Gateway ist:
+          <span :class="status">{{ status }}</span>
+        </p>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+
+import HealthService from "@/api/HealthService";
+import { useSnackbarStore } from "@/stores/snackbar";
+import HealthState from "@/types/HealthState";
+
+const snackbarStore = useSnackbarStore();
+const status = ref("DOWN");
+
+onMounted(() => {
+  HealthService.checkHealth()
+    .then((content: HealthState) => (status.value = content.status))
+    .catch((error) => {
+      snackbarStore.showMessage(error);
+    });
+});
+</script>
+
+<style scoped>
+.UP {
+  color: limegreen;
+}
+
+.DOWN {
+  color: lightcoral;
+}
```

Comment:
Naming CSS classes like this is generally bad practice (see BEM notation). We should consider refactoring this.

Reply:
#76